.github/ISSUE_TEMPLATE/suggestion.yml

```
@@ -0,0 +1,20 @@
+name: Suggestion
+description: Feel free to give us suggestions and enhancement ideas!
+labels: [enhancement]
+body:
+  - type: markdown
+    attributes:
+      value: "## Welcome!"
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking your time to give us suggestions! You gotta make sure these before reporting...
+        - This feature does not exist in the latest build.
+        - This suggestion wasn't already reported or being closed.
```

Wording can be improved here
```suggestion
        - This suggestion wasn't already reported or closed as not planned.
```